src/main/java/com/faforever/client/chat/UserInfoWindowController.java

```diff
@@ -145,7 +145,7 @@ private void plotPlayerStatistics(PlayerStatistics result) {
     List<XYChart.Data<Long, Integer>> values = new ArrayList<>();
 
     for (RatingInfo ratingInfo : result.getValues()) {
-      int minRating = RatingUtil.getRating(ratingInfo);
+      int minRating = RatingUtil.getGlobalRating(ratingInfo);
```

Comment:
`RatingInfo` is independent of global/leaderboard, so it should be `getRating()`